packages/cfd/src/Components/cfd-account-card.tsx

```diff
@@ -361,7 +361,7 @@ const CFDAccountCard = ({
                                 <Money
                                     amount={existing_data.display_balance}
                                     currency={existing_data.currency}
-                                    has_sign={existing_data.balance && existing_data.balance < 0}
+                                    has_sign={existing_data.balance ? existing_data.balance < 0 : false}
```

Comment:
has_sign was set number whereas it is expected to be boolean. Type conversion did not fix the issue so I had to make a condition to fix the warning.